src/platform/graphics/webgpu/webgpu-texture.js

```diff
@@ -384,6 +384,14 @@ class WebgpuTexture {
             if (anyUploads && anyLevelMissing && texture.mipmaps && !isCompressedPixelFormat(texture.format)) {
                 device.mipmapRenderer.generate(this);
             }
+
+            // update vram stats
+            if (texture._gpuSize) {
+                texture.adjustVramSizeTracking(device._vram, -texture._gpuSize);
+            }
+
+            texture._gpuSize = texture.gpuSize;
+            texture.adjustVramSizeTracking(device._vram, texture._gpuSize);
```

Comment:
Similar to https://github.com/playcanvas/engine/blob/4a28a45f9df1d01d36707b92d3fc8b9d904b5480/src/platform/graphics/webgl/webgl-texture.js#L630-L636